controllers/dspipeline_controller.go

```diff
@@ -231,9 +236,15 @@ func (r *DSPAReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 		if controllerutil.ContainsFinalizer(dspa, finalizerName) {
 			params.Name = dspa.Name
 			params.Namespace = dspa.Namespace
+			params.DSPONamespace = os.Getenv("DSPO_NAMESPACE")
```

Comment:
Could you modify `main.go` to exit if the environment variable `DSPO_NAMESPACE` isn't set since we require it?